src/dlgPackageExporter.cpp

```diff
@@ -499,19 +498,19 @@ void dlgPackageExporter::slot_export_package()
 
     if (!imageList.isEmpty()) {
         //Create description image dir
-        QString descriptionImageDirName = QStringLiteral("%1.mudlet/description_images/").arg(tempPath);
-        QDir descriptionImageDir = QDir(descriptionImageDirName);
+        QString descriptionImagesDirName = QStringLiteral("%1.mudlet/description_images/").arg(tempPath);
+        QDir descriptionImageDir = QDir(descriptionImagesDirName);
         if (!descriptionImageDir.exists()) {
-            descriptionImageDir.mkpath(descriptionImageDirName);
+            descriptionImageDir.mkpath(descriptionImagesDirName);
         }
         for (int i = imageList.size() - 1; i >= 0; i--) {
```

Comment:
Isn't there a nice C++14 `for (const auto <iterator> : <container> ) {...}` for this? :wink:

Reply:
I thought so too.

Reply:
Also not Part of this PR. New issue if it's worth discussing?